pkg/webhookadmission/webhook.go

```diff
@@ -122,13 +123,24 @@ func (c *Controller) processMutateRequest(w http.ResponseWriter, req *http.Reque
 		schedPath = podSpecSchedPath
 	}
 
+	// KUBEVIRT_SKIP_LD_PRELOAD_STATFS environment variable can be configured
+	// by the user as per their need through which stork can get to know if to
+	// actually create the configmap and mount the ld.so.preload and px_statfs.so
+	// files onto the virt-launcher container.
 	if !isStorkResource {
 		admissionResponse = &v1beta1.AdmissionResponse{
 			Result: &metav1.Status{
 				Message: "Ignoring backends which are not supported by stork ",
 			},
 			Allowed: true,
 		}
+	} else if c.KubevirtSkipPreloadStatFS {
```

Comment:
Is the the right place for the check? We will skip all the pod patches by checking it here. We just want to skip statfs patch.

Reply:
Made the changes to accomodate other patches. PTAL.